File: javaClient/src/main/java/cb/bots/RandomBot.java
package cb.bots;

import java.util.Random;

public class RandomBot implements BotAi {
    private String name;
    private int id;

    public RandomBot() {
        name = "JavaRandom";
        id = -1;
    }
    @Override
    public int play(int[][] field) {
        Random rand = new Random();
        return rand.nextInt(7);
    }

    @Override
    public String getName() {
        return name;
    }

    @Override
    public void setPlayerId(int playerId) {
        id = playerId;
    }

    @Override
    public int getPlayerId() {
        return id;
    }
}

File: csharpClient/Bots/RandomBot.cs
using CsClient.Bots;

namespace CsClient.Bots
{
    public class RandomBot : IBot
    {
        /// <inheritdoc/>
        public string Name { get; }

        /// <inheritdoc/>
        public int PlayerId { get; set; }

        /// <inheritdoc/>
        public int Play(int[][]? field)
        {
            Random r = new Random();
            return r.Next(7);
        }

        public RandomBot()
        {
            Name = "Random Bot";      
            PlayerId = -1;
        }
    }
}
